src/actions/index.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios'
+
+const ROOT_URL = 'http://localhost:8000'
+const FETCH_PRODUCTS = "fetch_products"
+const SAVE_CATEGORY = 'save_category'
+
+const fetchProducts = (pageNum = 1, category, sort) => {
+  let catQuery = ''
+  if (category) {
+    catQuery = `&category=${category}`
+  }
+  const req = axios.get(`${ROOT_URL}/products?page=${pageNum}${catQuery}&price=${sort}`)
```

Comment:
axios allows you to pass in parameters via an object that could make this cleaner